packages/apps/hr/src/components/molecules/ExpenseDetailsValidationButton/ExpenseDetailsValidationButton.js

```diff
@@ -57,8 +67,27 @@ const ExpenseDetailsValidationButton = ({expense, mode}) => {
     );
   }, [dispatch, mode, expense]);
 
+  const deleteExpenseLineAPI = useCallback(() => {
+    dispatch(deleteExpenseLine({ExpenseId: expense.id}));
+    navigation.navigate('ExpenseListScreen');
+  }, [dispatch, expense.id, navigation]);
+
   if (expense.statusSelect === Expense.statusSelect.Draft) {
-    return <Button title={I18n.t('Hr_Send')} onPress={sendExpenseAPI} />;
+    return (
+      <View style={styles.buttonContainer}>
+        <Button
+          title={I18n.t('Hr_Send')}
+          onPress={sendExpenseAPI}
+          width="45%"
+        />
+        <Button
+          title={I18n.t('Hr_Delete')}
+          onPress={deleteExpenseLineAPI}
+          width="45%"
+          color={Colors.errorColor}
+        />
```

Comment:
Maybe inverse order of buttons to match refuse and validate buttons (see below).

Reply:
And please add an icon on delete button